packages/ui-components/src/components/inline-editable-header/inline-editable-header.scss

```diff
@@ -0,0 +1,59 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/**
+	 * @prop --container-width: The width of the container;
+	 * @prop --container-height: The height of the container;
+	 * @prop --label-width: The width of the label;
+	 * @prop --label-height: The height of the label;
+	 * @prop --label-border: The label's border color;
+	 * @prop --label-border-radius: The border radius of the margin
+	 * @prop --label-color: The text color of the label
+	*/
+	--container-width: 508px;
+	--container-height: 36px;
+
+	--label-width: 416px;
+	--label-height: 32px;
+	--label-border: 1px solid #{kv-color('neutral-6')};
+	--label-border-radius: #{$spacing};
+
+	--label-color: #{kv-color('neutral-0')};
+}
+
+.inline-editable-header {
+	box-sizing: border-box;
+	width: var(--container-width);
+	height: var(--container-height);
+
+	.label {
+		@include ellipsis();
+		@include kv-font-h2-semibold;
+
+		cursor: pointer;
+		user-select: none;
+		width: var(--label-width);
+		height: var(--label-height);
```

Comment:
Do you really need it?

Reply:
the height is not necessary, I've gone ahead and changed it.